lib/project_types/script/cli.rb

```diff
@@ -5,44 +5,56 @@ class Project < ShopifyCli::ProjectType
     hidden_project_type
     creator 'Script', 'Script::Commands::Create'
 
+    register_command('Script::Commands::Deploy', 'deploy')
     register_command('Script::Commands::Test', 'test')
   end
 
   # define/autoload project specific Commads
   module Commands
     autoload :Create, Project.project_filepath('commands/create')
+    autoload :Deploy, Project.project_filepath('commands/deploy')
     autoload :Test, Project.project_filepath('commands/test')
   end
 
   # define/autoload project specific Forms
   module Forms
     autoload :Create, Project.project_filepath('forms/create')
+    autoload :Deploy, Project.project_filepath('forms/deploy')
+    autoload :ScriptForm, Project.project_filepath('forms/script_form')
   end
 
   module Layers
     module Application
+      autoload :BuildScript, Project.project_filepath('layers/application/build_script')
       autoload :CreateScript, Project.project_filepath('layers/application/create_script')
+      autoload :DeployScript, Project.project_filepath('layers/application/deploy_script')
       autoload :ExtensionPoints, Project.project_filepath('layers/application/extension_points')
       autoload :ProjectDependencies, Project.project_filepath('layers/application/project_dependencies')
       autoload :TestScript, Project.project_filepath('layers/application/test_script')
     end
 
     module Domain
       autoload :Errors, Project.project_filepath('layers/domain/errors')
+      autoload :DeployPackage, Project.project_filepath('layers/domain/deploy_package')
```

Comment:
I ❤️ how you are keeping these alphabetical!  I also noticed the only exception is `Error`.  👍  Awesome!!